Library/Homebrew/unversioned_cask_checker.rb

```diff
@@ -83,8 +88,8 @@ def all_versions
 
         installer.extract_primary_container(to: dir)
 
-        info_plist_paths = (apps + qlplugins).flat_map do |app|
-          top_level_info_plists(Pathname.glob(dir/"**"/app.source.basename/"Contents"/"Info.plist")).sort
+        info_plist_paths = (apps.concat(qlplugins)).flat_map do |artifact|
```

Comment:
@maxim-belkin The previous iteration was failing the typecheck. This change resolves it. Or I noticed that using `[] + apps + qlplugins` also works. What is the best was to handle it?

Reply:
`.concat` is good! thank you!